app/tokens.json

```diff
@@ -0,0 +1,5 @@
+{
+  "FVM": {
+    "address": ""
+  }
```

Comment:
dont think we need FVM here

Reply:
I used to tell what name of the native coin Kub

Reply:
okay

Reply:
I moved native token config to env file kub